src/Parser/RetributionPaladin/Modules/Features/Judgment.js

```diff
@@ -61,7 +62,7 @@ class Judgment extends Module {
 		const unbuffedJudgmentPercentage = this.spenderOutsideJudgment / this.totalSpender;
 		when(unbuffedJudgmentPercentage).isGreaterThan(0.05)
 			.addSuggestion((suggest,actual,recommended) => {
-				return suggest('You\'re spending Holy Power outisde of the Judgment debuff Window too much. Only spending while the enemy has Judgment on them is very important.')
+				return suggest(<span>You're spending Holy Power outisde of the <SpellLink id={SPELLS.JUDGMENT_CAST.id} /> debuff Window too much. Only spending while the enemy has the debuff on them is very important.</span>)
```

Comment:
Just need to clean up the language here. I get the underlying point but potentially reword this to stay something like:
`You're spending Holy Power outside of the Judgment debuff. It is optimal to only spend Holy Power while the enemy is debuffed with Judgement.`